src/kademlia/dht_tracker.cpp

```diff
@@ -208,19 +215,19 @@ namespace libtorrent { namespace dht
 #ifndef TORRENT_NO_DEPRECATE
 	void dht_tracker::dht_status(session_status& s)
 	{
-		m_dht.status(s);
+		m_dht.status(s); //TODO: What to do with m_dht6?
```

Comment:
also not for this patch, but I imagine that the status() call should be strictly additive, and we would call it once for each node